app/src/main/java/app/revanced/manager/MainActivity.kt

```diff
@@ -52,8 +74,17 @@ class MainActivity : ComponentActivity() {
                 NavBackHandler(navController)
 
                 val showAutoUpdatesDialog by vm.prefs.showAutoUpdatesDialog.getAsState()
+
                 if (showAutoUpdatesDialog) {
-                    AutoUpdatesDialog(vm::applyAutoUpdatePrefs)
+                    rememberLauncherForActivityResult = rememberSaveable { mutableStateOf(LegacyActivity.NOT_LAUNCHED) }
```

Comment:
I was referring to this: https://developer.android.com/reference/kotlin/androidx/activity/compose/package-summary#rememberLauncherForActivityResult(androidx.activity.result.contract.ActivityResultContract,kotlin.Function1)